conf/sample.ini

```diff
@@ -811,3 +811,6 @@
 
 # Experimental feature
 ;use_browser_locale = false
+
+# Default timezone for user preferences. Options are 'browser' for the browser local timezone or a timezone name from IANA Time Zone database
```

Comment:
Would be good with a couple examples here to understand what format is expected.

```suggestion
# Default timezone for user preferences. Options are 'browser' for the browser local timezone or a timezone name from IANA Time Zone database, e.g. UTC or Europe/Amsterdam etc.
```